apis/keda/v1alpha1/scaledobject_webhook.go

```diff
@@ -231,8 +231,8 @@ func verifyHpas(incomingSo *ScaledObject, action string, _ bool) error {
 		return err
 	}
 
-	var incomingSoGckr GroupVersionKindResource
-	incomingSoGckr, err = ParseGVKR(restMapper, incomingSo.Spec.ScaleTargetRef.APIVersion, incomingSo.Spec.ScaleTargetRef.Kind)
+	var incomingSvGvkr GroupVersionKindResource
+	incomingSvGvkr, err = ParseGVKR(restMapper, incomingSo.Spec.ScaleTargetRef.APIVersion, incomingSo.Spec.ScaleTargetRef.Kind)
```

Comment:
why Sv instead of So?

Reply:
> why Sv instead of So?

Because GVKR represents GroupVersionKindResource, and this is be seen more common and explicit.

Reply:
Yeah, `Gckr` vs `Gvkr` is clear (and the change is 100% worth), but `SO` means `ScaledObject`, that's why my question

Reply:
> Yeah, `Gckr` vs `Gvkr` is clear (and the change is 100% worth), but `SO` means `ScaledObject`, that's why my question

Haha, an oolong，I have revised. Wait for your review. thanks